### PR TITLE
Add test for stats/update.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmrepo/32/
Test PASSed.

**Comment by urosj** [comment]
LGTM

File: csclient/csclient_test.go [hunk 1]
```diff
@@ -304,16 +304,16 @@ func (s *suite) TestGetArchiveWithStatsDisabled(c *gc.C) {
...
-//}
+func (s *suite) TestStatsUpdate(c *gc.C) {
+	key := s.checkGetArchive(c)
+	s.checkCharmDownloads(c, key, 1)
+	err := s.client.StatsUpdate(params.StatsUpdateRequest{
+		CharmReference: charm.MustParseReference("~charmers/utopic/wordpress-42"),
+		Timestamp: time.Now(),
+		Type: params.Deploy})
+	c.Assert(err, gc.IsNil)
+	s.checkCharmDownloads(c, key, 2)
```
Comment (urosj) [inline csclient/csclient_test.go]: Add a note that this is an error and a TODO for fixing this, so this test will fail in the future.

**Comment by mhilton** [comment]
:+1:

**Comment by fabricematrat** [comment]
:shipit: thanks @mhilton  && @urosj

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmrepo-merge

**Pull request merged**